Vagrantfile

```diff
@@ -0,0 +1,11 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "bento/ubuntu-20.04"
```

Comment:
Using Ubuntu will limit our ability to assist you with system issues. Please consider using Fedora or CentOS.

Reply:
But as discussed in Slack, wouldn't it be better to choose a more "out of the box" box that GitHub actions support?
Correct me if I'm wrong, using Docker to run Fedora won't replace the usage of Vagrant?

Reply:
Don't confuse GitHub actions and Vagrant. There is a an up to date Fedora Vagrant box that is officially released and supported by the Fedora project. For Vagrant you should use that.

With github actions - if you use the GitHub provided VMs - they are pre-installed with some Ubuntu version simply because that's what the GH engineers chose to provision their AZURE VMs with - but they've included the container support feature precisely to make that choice meaningless. 

As Red Hatters we work with Fedora, RHEL and CentOS all the time, we will be much better equipped to help you solve issues that arise on a Fedora environment then if those issue arise on Ubuntu or some other environment.

Reply:
Ok then, Im moving back to fedora/32-cloud-base